react/features/participants-pane/components/breakout-rooms/components/web/RoomParticipantContextMenu.tsx

```diff
@@ -0,0 +1,116 @@
+import { makeStyles } from '@material-ui/core';
+import React, { useCallback, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+
+// @ts-ignore
+import { Avatar } from '../../../../../base/avatar';
+// @ts-ignore
+import { ContextMenu, ContextMenuItemGroup } from '../../../../../base/components';
+// @ts-ignore
+import { isLocalParticipantModerator } from '../../../../../base/participants';
+// @ts-ignore
+import { getBreakoutRooms } from '../../../../../breakout-rooms/functions';
+// @ts-ignore
+import { showOverflowDrawer } from '../../../../../toolbox/functions.web';
+// @ts-ignore
+import SendToRoomButton from '../../../../../video-menu/components/web/SendToRoomButton';
+
+type Props = {
+
+    /**
+     * Room and participant jid reference.
+     */
+    entity: {
+        room: any,
+        jid: string,
+        participantName: string
+    },
+
+    /**
+     * Target elements against which positioning calculations are made.
+     */
+    offsetTarget: HTMLElement|undefined,
+
+    /**
+     * Callback for the mouse entering the component.
+     */
+    onEnter: Function,
+
+    /**
+     * Callback for the mouse leaving the component.
+     */
+    onLeave: Function,
+
+    /**
+     * Callback for making a selection in the menu.
+     */
+    onSelect: Function
+};
+
+const useStyles = makeStyles((theme:any) => {
+    return {
+        text: {
+            color: theme.palette.text02,
+            padding: '10px 16px',
+            height: '40px',
+            overflow: 'hidden',
+            display: 'flex',
+            alignItems: 'center',
+            boxSizing: 'border-box'
+        }
+    };
+});
+
+export const RoomParticipantContextMenu = ({
+    entity,
+    offsetTarget,
+    onEnter,
+    onLeave,
+    onSelect
+}: Props) => {
+    const styles = useStyles();
+    const { t } = useTranslation();
+    const isLocalModerator = useSelector(isLocalParticipantModerator);
+    const lowerMenu = useCallback(() => onSelect(true), [onSelect]);
+    const rooms:any = useSelector(getBreakoutRooms);
```

Comment:
Why is the any annotation necessary here?

Reply:
Changed it to Object